pkgs/applications/misc/chataigne/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, fetchurl, appimageTools, pkgs }:
+
+let
+  pname = "chataigne";
+  version = "1.8.0";
+  name = "${pname}-${version}";
+  src = fetchurl {
+    url = "https://benjamin.kuperberg.fr/chataigne/user/data/Chataigne-linux-x64-1.8.0.AppImage";
+    name="${pname}-${version}.AppImage";
```

Comment:
Could you run `nixpkgs-fmt` to fix this? Off the top of my head:

    nix-shell -p nixpkgs-fmt -I nixpkgs=~/src/nixpkgs   # or similar
    nixpkgs-fmt FILENAME